experimental/generated/elasticsearch/7/template.json

```diff
@@ -1,11 +1,11 @@
 {
+  "_meta": {
+    "version": "1.7.0+exp"
+  },
```

Comment:
@ebeahan Alright I took some liberties on where innocuous metadata was nested in this PR.

The 1.7 release (and all previous ones) incorrectly had the `_meta` fields inside the mappings. I wonder if I should modify this backport to not change this 🤔 

What do you think?

Reply:
I tried testing out these changes to the legacy template, but `_meta` doesn't appear supported as a the top-level field in the legacy index template API. Trying to upload the template with `_meta` present resulted in this error:

```json
{
  "error": {
    "root_cause": [
      {
        "type": "parse_exception",
        "reason": "unknown key [_meta] in the template "
      }
    ],
    "type": "parse_exception",
    "reason": "unknown key [_meta] in the template "
  },
  "status": 400
}
```

Reply:
Oh thanks for testing that 👍 I should totally have done that! 

But this means I'll have to fix that in master as well.